### PR TITLE
Update the linux conda environment

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
>Do you know why the old environment was incorrect?

Not sure. I guess we wanted to test with different/latest llvm version?

File: environment_linux.yml [hunk 1]
```diff
@@ -19,3 +19,6 @@ dependencies:
   - rapidjson
   - ninja
   - zlib
+  - zstd-static
+  - cmake
+  - make
```
Comment (Shaikh-Ubaid) [inline environment_linux.yml:24]: We should pin these to exact versions to avoid any failure in future.

**Review by Shaikh-Ubaid — APPROVED** [review]
Great catch! Thanks for this! I shared a minor comment. Rest looks good to me.

**Review by czgdp1807 — APPROVED** [review]

File: environment_linux.yml (hunk 1 above)
Comment (certik) [reply environment_linux.yml:24]: We should pin all of them, in the past we had errors due to the json package, etc. We'll do it later, this way at least we have something that is close.

**Pull request merged**